### DISCUSSION TITLE
Focus on fully translating the platform into other languages.

### DISCUSSION THREAD

**Discussion by megarockss** [linuxdeepin/developer-center#6628] [Features Request & Ideas | 特性请求 & 头脑风暴]
The platform's lack of translation in some areas limits its use to many people who would like to use the platform daily. When I discovered the first versions of deepin, they were much more attentive to this. The platform's qualifications have been falling rapidly, I ask that you not only focus on the Chinese people, but on everyone who supports the platform. I'm Brazilian, and I really appreciate the structure of this beautiful distribution. Please don't forget us.

**Comment by BLumia** [comment]
Hi and thank you for letting us know this issue, for the missing-translation issue, are you using v20.9 or v23 beta? Could you provide some detailed information about which application and which part is not fully translated (screenshot could be good), and additionally, would you like to join the translation SIG and contribute on translation?

Thanks!
